dipy/workflows/io.py

```diff
@@ -27,13 +27,12 @@ def run(self, input_files,
         input_files : variable string
             Any number of Nifti1, bvals or bvecs files.
         b0_threshold : float, optional
-            (default 50)
+            Threshold used to find b=0 directions.
```

Comment:
b=0 images have no diffusion encoding gradients, and thus the `direction` does no apply. It could be something like:
```Threshold used to find b0 images.```
or 
```Threshold used to find b0 volumes.```

Reply:
Addressed in e5af17f